debian/control.top.in

```diff
@@ -34,6 +34,7 @@ Build-Depends:
     libusb-1.0-0-dev,
     libxmu-dev,
     netcat-traditional | netcat-openbsd | netcat,
+    netpbm,
```

Comment:
Is it correct that this belong here?

Reply:
This is meant to help avoiding the warnings like the following for the HTML documentation
```
Formatting carousel.9 as HTML
Formatting bldc.9 as HTML
troff:./es/man9/bldc.9:462: error: suppression limit registers span more than a page; grohtml-info for image 1 will be wrong
sh: 1: psselect: not found
Calling 'psselect -q -p7 /tmp/groff-ps-TjNd7Q /tmp/groff-ps-FXUVKC
' returned status 32512
sh: 1: pnmtopng: not found
sh: 1: pnmcut: not found
sh: 1: pnmcrop: not found
Calling 'pnmcut 200 102 551 908 < /tmp/groff-page-RmSE2N | pnmcrop -quiet | pnmtopng -quiet -background rgb:f/f/f -transparent rgb:f/f/f> grohtml-66692-1.png
' returned status 32512
Formatting charge_pump.9 as HTML
Formatting clarke2.9 as HTML
```
I am not ultimately sure why I have not placed it next to the build-depends-indep, maybe because the man pages are also built for our architecture-dependent packages? But I have also looked again in the build log for Debian packages and have not found that warning that I had seen for the HTML docs target. Hm.

So, admittedly, I am now no longer ultimately sure that we indeed need this for the Debian packages, but then again, the htmldocs build target gets its dependencies from the regular package's dependencies as shown in https://github.com/LinuxCNC/linuxcnc/blob/master/scripts/travis-install-build-deps.sh. So, I would just add it, maybe shift it to build-depends-indep, and maybe with the <!nodocs> addition?